src/resources/triton_python_backend_utils.py

```diff
@@ -295,6 +295,128 @@ def triton_string_to_numpy(triton_type_string):
     return TRITON_STRING_TO_NUMPY[triton_type_string]
 
 
+def set_max_batch_size(config, value):
+    """Set the max batch size for the model
+    Parameters
+    ----------
+    config : AutoCompleteConfig object
+        object containing the model configuration
+    value : int
+        the max_batch_size we want to set for the model
+    """
+    config.set_max_batch_size(value)
+
+
+def set_input(config, new_input):
+    """Set the input for the model
+    Parameters
+    ----------
+    config : AutoCompleteConfig object
+        object containing the model configuration
+    new_input : dict
+        containing the input we want to set for the model
+    Raises
+    ------
+    ValueError
+        If 'new_input' contains property other than 'name',
+        'data_type' and 'dims' or any of the properties is
```

Comment:
.. are not set.

Do we have tests for missing properties?

Reply:
Tests are added.